IWXXM/examples/metar-EDDF-runwaystate.xml

```diff
@@ -23,7 +23,7 @@
                     <aixm:ARP>
                         <aixm:ElevatedPoint axisLabels="Lat Long"
                             gml:id="uuid.7e308d55-3419-4169-8861-2a88af3b90ac" srsDimension="2"
-                            srsName="https://www.opengis.net/def/crs/EPSG/0/4326">
+                            srsName="http://www.opengis.net/def/crs/EPSG/0/4326">
```

Comment:
Is this change to srsName values necessary for successful validation? 
Or:
- You intend to recommend that "http" be used throughout document?
- Playing it safe?

Reply:
Just want to make it have a consistent look.  No particular reason.  Feel free to revert.